### PR TITLE
Exposing the last action in the Agent API

### PR DESCRIPTION
**Pull request by vincentpierre**

### CODE REVIEW

File: com.unity.ml-agents/Runtime/Agent.cs [hunk 1]
```diff
@@ -689,6 +689,14 @@ public virtual void AgentReset()
         {
         }
 
+		/// <summary>
+        /// Returns the last action that was decided on by the Agent (returns null if no decision has been made)
+        /// </summary>
+		public float[] GetLastAction()
```
Comment (surfnerd) [inline com.unity.ml-agents/Runtime/Agent.cs]: here too

File: com.unity.ml-agents/Runtime/Agent.cs [hunk 2]
```diff
@@ -689,6 +689,14 @@ public virtual void AgentReset()
         {
         }
 
+		/// <summary>
```
Comment (surfnerd) [inline com.unity.ml-agents/Runtime/Agent.cs]: nit: spacing looks off.  Look like a `\t` was added.

**Comment by vincentpierre** [comment]
Are we okay with adding this function to the public API and maintain it ?

**Comment by chriselion** [comment]
I'm OK having it in the API. Wondering if `GetAction` or `GetActions` would be a better name though?

**Pull request merged**